source/operator.rst

```diff
@@ -528,7 +528,10 @@ The replsets section controls the MongoDB Replica Set.
 | **Description** | The `Kubernetes Storage Class								|
 |                 | <https://kubernetes.io/docs/concepts/storage/storage-classes/>`_				|
 |                 | to use with the MongoDB container `Persistent Volume Claim 					|
-|                 | <https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims>`_	|
+|                 | <https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims>`_.	|
```

Comment:
can we also add this to System Requirements?

Reply:
done in https://github.com/percona/percona-server-mongodb-operator/pull/342